backend/api/views/simplePostView.py

```diff
@@ -37,17 +26,12 @@ def createPost(request, author_id):
 #
 # return query result in json list
 ################################################
-@api_view(['GET'])
-def getPost(request, author_id, post_id):
-  data = post.Post.objects.all()
-  # try to filter for such post
+@api_view(['GET', 'PUT'])
```

Comment:
this should probably be separate functions for GET and PUT because they are likely to be handled differently